properties/service_fabric_common.props

```diff
@@ -12,7 +12,10 @@
     
     <!-- Set Drop folders -->
     <DropFolder>$(RepoRoot)drop\$(Configuration)\</DropFolder>
-    <DropFolderNetStandard>$(RepoRoot)drop\$(Configuration)\netstandard\</DropFolderNetStandard>
+    <DropFolderNetFramework>$(RepoRoot)drop\$(Configuration)\net461\</DropFolderNetFramework>
```

Comment:
We dont need other framworks here?

Reply:
Not required to create separate drop folders for each framework. The dlls in net461 drop folder will be copied to other net framework libs and runtimes folders while creating nupkg.